src/components/post-ctrls/PostCtrls.tsx

```diff
@@ -295,12 +240,11 @@ let PostCtrls = ({
         />
       </View>
       <View style={big ? a.align_center : [a.flex_1, a.align_start]}>
-        <Pressable
+        <PostCtrlButton
           testID="likeBtn"
-          style={btnStyle}
+          big={big}
           onPress={() => requireAuth(() => onPressToggleLike())}
-          accessibilityRole="button"
-          accessibilityLabel={
+          label={
             post.viewer?.like
               ? _(
                   msg`Unlike (${plural(post.likeCount || 0, {
```

Comment:
```suggestion
          label={
            post.viewer?.like
              ? _(
                  msg({
                    message: `Unlike (${plural(post.likeCount || 0, {
                      one: '# like',
                      other: '# likes',
                    })})`,
                    comment: 'Accessibility label for the like button when the post has been liked, verb form followed by number of likes and noun form',
                  })
                )
              : _(
                  msg({
                    message: `Like (${plural(post.likeCount || 0, {
                      one: '# like',
                      other: '# likes',
                    })})`,
                    comment: 'Accessibility label for the like button when the post has not been liked, verb form followed by number of likes and noun form',
                  })
                )
          }
```
Sorry, I couldn't select all the current code here...